concourse/Dockerfile

```diff
@@ -7,6 +7,11 @@ RUN apt-get update --fix-missing && apt-get -y upgrade \
       postgresql-11 redis-server \
     && rm ./google-chrome-stable_current_amd64.deb
 
+RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
```

Comment:
Do we also want to run `yarn install` here, like we do `bundle install` further down?  And then add a trigger for `build-tests-image` based on the yarn lock changing?

Reply:
I've added a `yarn install` here too.  I don't think we'll need to add any triggers for `build-tests-image`.

Reply:
I think the `yarn install` will have to come after `COPY`ing some stuff in, eg

```
RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
    && apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 23E7166788B63E1E \
    && apt update -y \
    && apt install -y yarn

COPY Gemfile* .ruby-version package.json /application/

WORKDIR /application/
RUN yarn install
RUN bundle install
```

Reply:
Thanks, I didn't spot we weren't already in the application directory at that point